src/explore-education-statistics-admin/src/pages/release/datablocks/components/__tests__/ChartDataSetsAddForm.test.tsx

```diff
@@ -114,7 +109,7 @@ describe('ChartDataSetsAddForm', () => {
 
     const locationOptions = within(location).getAllByRole('option');
     expect(locationOptions).toHaveLength(3);
-    expect(locationOptions[0]).toHaveTextContent('Any location');
+    expect(locationOptions[0]).toHaveTextContent('All location');
```

Comment:
Should that read "All locations" with an "s"?

Reply:
Typo indeed!